### ISSUE TITLE
Add ability to produce non-salmon harvest estimates

### ISSUE THREAD

**Issue by bstaton1** [bstaton1/KuskoHarvEst#116]
- [x] Keep whitefish/sheefish columns in `prepare_interviews_one()`
- [x] Include calculation in `estimate_catch_per_trip()`
- [x] Include calculation in `estimate_harvest()`
- [x] ~~Include option to accept this species (potentially plural) in `report()`~~
- [x] Create a table that can summarize non-salmon harvest for placement in the appendix
- [x] Create option to produce whitefish harvest estimates in `report_tool()` and `build_rmd()`

I may be missing a necessary change here...

**Comment by bstaton1** [comment]
The documents can now report nonsalmon harvest. Here is code for a non-salmon appendix:

~~~~~~~~~
\clearpage
\setcounter{figure}{0}
\setcounter{table}{0}
\renewcommand\thefigure{B\arabic{figure}}
\renewcommand\thetable{B\arabic{table}}

# Appendix: Non-salmon Harvest Information {-}

```{r estimate-harvest-nonsalmon}
# estimate (via bootstrap) drift net harvest for all species and and strata
boot_out_drift = bootstrap_harvest(interview_data, drift_effort_info,
                                   gear = "drift", n_boot = n_boot, stratify_interviews = TRUE, nonsalmon = TRUE)

# estimate (via bootstrap) set net harvest for all species and and strata
boot_out_set = bootstrap_harvest(interview_data, set_effort_info,
                                 gear = "set", n_boot = n_boot, stratify_interviews = FALSE, nonsalmon = TRUE)

# sum up across gears
boot_out_total = data.frame(iter = boot_out_drift$iter, 
                            date = boot_out_drift$date,
                            gear = "total", 
                            stratum = boot_out_drift$stratum,
                            sheefish = rowSums(cbind(boot_out_drift[,"sheefish"], boot_out_set[,"sheefish"]), na.rm = TRUE),
                            whitefish = rowSums(cbind(boot_out_drift[,"whitefish"], boot_out_set[,"whitefish"]), na.rm = TRUE),
                            total = rowSums(cbind(boot_out_drift[,"total"], boot_out_set[,"total"]), na.rm = TRUE)
)

# combine drift, set, and total into one data frame of bootstrap output
boot_out <<- rbind(boot_out_total, boot_out_drift, boot_out_set)
```

```{r set-summaries-nonsalmon}
set_harvest = sapply(c("sheefish", "whitefish", "total"), function(sp) report(sp, gear = "set", CI = FALSE, return_numeric = TRUE))
set_spp = KuskoHarvEst:::percentize(set_harvest[c("sheefish", "whitefish")]/set_harvest["total"])
```

* An estimated total of **`r report()`** non-salmon (sheefish and whitefishes) were harvested.
  * An estimated total of **`r report(spp = "sheefish")`** sheefish were harvested.
  * An estimated total of **`r report(spp = "whitefish")`** whitefishes were harvested.
* Harvest by set nets accounted for an estimated **`r report(gear = "set")`** total non-salmon fishes (`r paste0(set_spp["sheefish"], " sheefish and ", set_spp["whitefish"], " whitefishes")`).

```{r}
make_strata_summary_table(interview_data, "drift", nonsalmon = TRUE)
```

```{r}
make_appendix_table(interview_data, "drift", "sheefish")
```

```{r}
make_appendix_table(interview_data, "drift", "whitefish")
```
~~~~~~~~~

I'm waiting to produce Rmd script files and interface components for this until I receive feedback from the group.

**Comment by bstaton1** [comment]
Closed by #160.
